### PR TITLE
HAWQ-1229. Remove verbose option in hawq config

### PR DESCRIPTION
**Pull request by radarwave**

### CODE REVIEW

**Comment by paul-guo-** [comment]
'-v' seem to have another usage in "hawq config"?

[-v hawq_property_value | --value=hawq_property_value

**Comment by radarwave** [comment]
Yeah, '-v' is used to set the GUC value. '--verbose' is a legacy option, now it's already been removed. This time only remove the wrong help message.

**Comment by paul-guo-** [comment]
I mean why not replace this time.

**Comment by radarwave** [comment]
Good point, I will update it.

**Comment by radarwave** [comment]
@paul-guo- Please review. Thanks.

File: tools/bin/hawqpylib/HAWQ_HELP.py [hunk 1]
```diff
@@ -159,10 +159,10 @@
 
 The "options" are:
    -c --change         Changes a configuration parameter setting.
```
Comment (paul-guo-) [inline tools/bin/hawqpylib/HAWQ_HELP.py]: Actually I think "--change" could better be "--parameter"? ". How do you think?

Comment (radarwave) [inline tools/bin/hawqpylib/HAWQ_HELP.py]: We'd better not change it, HAWQ have many client option names from GPDB, sometimes it's good to have similar options for the users who use both. And if we change any options, we have to change it together with Ambari.

Comment (paul-guo-) [inline tools/bin/hawqpylib/HAWQ_HELP.py]: How. Then +1.

**Pull request merged**